src/redist/targets/Versions.targets

```diff
@@ -33,8 +33,7 @@
       <CliVersionPrefix>$(CliVersionNoSuffix)-$(ReleaseSuffix)</CliVersionPrefix>
       <CliBrandingVersion Condition=" '$(DropSuffix)' != 'true' ">$(CliVersionNoSuffix) - $(ReleaseSuffix)</CliBrandingVersion>
       <CliBrandingVersion Condition=" '$(DropSuffix)' == 'true' ">$(CliVersionNoSuffix)</CliBrandingVersion>
-      <SimpleVersion Condition=" '$(DropSuffix)' != 'true' ">$(CliVersionNoSuffix).$(GitCommitCount)</SimpleVersion>
-      <SimpleVersion Condition=" '$(SimpleVersion)' == '' ">$(CliVersionNoSuffix)</SimpleVersion>
+      <SDKBundleVersion>$(CliVersionNoSuffix).$(GitCommitCount)</SDKBundleVersion>
```

Comment:
I assume the GitCommitCount won't exceed 65535? Version numbers are capped as unsigned 16-bit integers

Reply:
At some point it will. We're at 10536 and going + 1 for each commit. :( 

We have been talking about moving away from commit height for a while and I think we can now. Arcade should provide us with a 16-bit safe file version number IIRC. 

Since we've already used commit height for 3.0.100.*, perhaps we should wait for 3.0.200+ to start before changing it. We have 50,000 commits to solve this...

Reply:
This sounds like a plan to me.